src/content/posts/how-to-raise-funds/index.md

```diff
@@ -0,0 +1,135 @@
+
+### Step 1: Fill in Project Metadata
+
+- **Navigate to the "Create Project" Section**: Navigate to the 'Founder' section and click on 'Create Project'.
+
+- **Enter Project Metadata**:
+  - **Project Name**: Enter a clear and descriptive name for your project.
+  - **About**: Provide detailed information about your project, including its objectives and vision.
+  - **Project Website**: Enter the URL of your project’s website, if available.
+  - **Banner**: Enter the URL of your project’s banner image to make your project page visually appealing.
+  - **Nip 05 and Nip 57 (zaps)**: Fill these fields with the relevant information if applicable.
+  - **Image**: Enter the URL of your project’s reference image.
+
+    ![Project Metadata](./assets/project-metadata.svg)
+
+  - Click "Next" to proceed to the next step.
+
+### Step 2: Provide Project Info
+
+- **Enter Project Identifier and Founder Key**: These fields will be auto-generated by Angor.
+- **Start Date and Expiry Date**: Set the start date and the expiry date for your project.
+- **Penalty Days**: Define the number of penalty days.
+- **Target Amount**: Specify the total amount of funding you aim to raise.
+- **Define Project Stages**:
+  - Allocate a percentage of the total funds to each stage.
+  - Set the date for each stage.
+  - Click "Next" to proceed to the next step.
+
+    ![Project Info](./assets/project-info.svg)
+
+### Step 3: On-Chain Confirmation
+
+- **Review Project Details**: Review all the details of your project.
+- **Confirm Project Creation**: Once all the information is verified, click on "Submit" to finalize the creation of your project on the blockchain.
+
+    ![On-Chain Confirmation](./assets/on-chain.svg)
+
+
+### Proceeding to the Next Section
+After filling out all the required information in the Project Info section, click the "Next" button to move on to the On Chain section, where you will define blockchain-related parameters for your project.
+
+![Example Project View](./assets/view-project.svg)
+
+### Post Project Updates on Nostr
+
+- Export the private key from Angor.
+- Import the private key into a Nostr client.
+- Post updates on project progress and milestone completion on Nostr.
+- Ensure updates are clear and informative for investors.
+
+### Spend Funds for Milestones
+
+- As a founder, once a milestone is reached, sign the transaction to spend the funds for that milestone.
+- Ensure the spending aligns with the milestone requirements and project goals.
+
+  ![Founder stage funds claim](./assets/founder-stage-claim.svg)
+
+### Claiming Funds 
+
+As a founder on Angor involves reaching and verifying milestones, providing updates, and then releasing the funds tied to those milestones through time-lock contracts. This ensures that the project progresses in a structured and accountable manner, protecting both the founder and the investors.
+
+#### 1. Monitor Project Progress
```

Comment:
this are unrelated to the founder it is a bout the investor right?

Reply:
Added such that being a investor is also like a founder but I will sort that out